mlflow/sklearn/__init__.py

```diff
@@ -531,6 +531,13 @@ def autolog():
       **Metrics**
         - A training score obtained by ``estimator.score``. Note that the training score is
           computed using parameters given to ``fit()``.
+        - Common metrics for classifier:
+          (1) precision score. (2) recall score. (3) f1_score. (4) accuracy score.
+          If the classifier has method `predict_proba`, we additionally log:
+          (5) log loss. (6) roc_auc_score
```

Comment:
Can we use a nested list here?

![Screen Shot 2020-09-01 at 9 24 16](https://user-images.githubusercontent.com/17039389/91781323-05072e80-ec35-11ea-841a-0a2bb13bc92d.png)

code
```
          - precision score
          - recall score
          - f1 score
          - accuracy score

          If the classifier has method ``predict_proba``, we additionally log:

          - log loss
          - roc auc score
```

Reply:
and can we add hyperlinks to these metrics to clarify which function we use to compute them? For example,

```
- `presicion score`_

.. _precision score:
    https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html
```